core/src/main/scala/kafka/server/BrokerFeatures.scala

```diff
@@ -72,6 +73,12 @@ class BrokerFeatures private (@volatile var supportedFeatures: Features[Supporte
 object BrokerFeatures extends Logging {
 
   def createDefault(): BrokerFeatures = {
+    new BrokerFeatures(Features.supportedFeatures(
+      java.util.Collections.singletonMap(MetadataVersion.FEATURE_NAME,
```

Comment:
Maybe this is out of scope for this PR, but it seems like the `BrokerFeatures` class just duplicates `FinalizedControllerFeatures`. Why not just use `FinalizedControllerFeatures` instead in the old controller?